pyg_lib/ops/__init__.py

```diff
@@ -223,12 +223,40 @@ def sampled_div(
     return out
 
 
+def index_sort(input: LongTensor,
+               max: Optional[int] = None) -> Tuple[LongTensor, LongTensor]:
+    r"""Sorts the elements of the :obj:`input` tensor in ascending order by
+    value. It is expected that :obj:`input` tensor is 1-dimensional and
+    contains only positive, integer values. If :obj:`max` is given, it can be
+    used by the underlying algorithm for better performance.
+
+    .. note::
+
+        This operation works only for tensors associated with the CPU device.
```

Comment:
Can we add a simple fallback here for GPU, e.g, just using regular `torch.sort`?

Reply:
Yes, sure.